shell/platform/darwin/ios/framework/Source/accessibility_bridge_test.mm

```diff
@@ -5,11 +5,61 @@
 #import <XCTest/XCTest.h>
 
 #import "flutter/shell/platform/darwin/common/framework/Headers/FlutterMacros.h"
+#import "flutter/shell/platform/darwin/ios/framework/Headers/FlutterPlatformViews.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h"
 #import "flutter/shell/platform/darwin/ios/framework/Source/accessibility_bridge.h"
 #import "flutter/shell/platform/darwin/ios/platform_view_ios.h"
 #import "third_party/ocmock/Source/OCMock/OCMock.h"
 
 FLUTTER_ASSERT_NOT_ARC
+@class MockPlatformView;
+MockPlatformView* mockPlatformView = nil;
+
+@interface MockPlatformView : UIView
+@end
+@implementation MockPlatformView
+
+- (void)dealloc {
+  mockPlatformView = nil;
+  [super dealloc];
+}
+
+@end
+
+@interface MockFlutterPlatformView : NSObject <FlutterPlatformView>
+@property(nonatomic, strong) UIView* view;
+@end
+
+@implementation MockFlutterPlatformView
+
+- (instancetype)init {
+  if (self = [super init]) {
+    MockPlatformView* view = [[MockPlatformView new] autorelease];
+    mockPlatformView = view;
+    self.view = view;
+  }
+  return self;
+}
+
+- (void)dealloc {
+  self.view = nil;
+  [super dealloc];
+}
+
+@end
+
+@interface MockFlutterPlatformFactory : NSObject <FlutterPlatformViewFactory>
+@end
+
+@implementation MockFlutterPlatformFactory
+- (NSObject<FlutterPlatformView>*)createWithFrame:(CGRect)frame
+                                   viewIdentifier:(int64_t)viewId
+                                        arguments:(id _Nullable)args {
+  MockFlutterPlatformView* platformView = [[MockFlutterPlatformView new] autorelease];
```

Comment:
Your example is using autorelease (which generally is proper) but other plugins are not doing this: https://github.com/flutter/plugins/blob/531ff2bc80afb3181218a1ff46e195303348ae0c/packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapController.m#L37

Reply:
done

Reply:
@gaaclarke Emm, I think they are the same thing, they also return autorelease object actually by ARC.

Reply:
You're right.  Sorry, half of our codebase is in ARC half is not.  I thought I was looking at non-arc code.